pywinauto/unittests/test_uiawrapper.py

```diff
@@ -1419,6 +1420,50 @@ def test_button_click(self):
             itm = lst_ctl.children()[1]
             self.assertEqual(itm.texts()[0], u'Red')
 
+    class ToolbarMfcTests(unittest.TestCase):
+
+        """Unit tests for ToolbarWrapper class on MCF demo"""
+
+        def setUp(self):
+            """Set some data and ensure the application is in the state we want"""
+            _set_timings()
+
+            # start the application
+            self.app = Application(backend='uia')
+            self.app = self.app.start(mfc_app_rebar_test)
+            self.dlg = self.app.RebarTest
+            self.tb = self.dlg.MenuBar.wrapper_object()
+
+        def tearDown(self):
+            """Close the application after tests"""
+            self.app.kill()
```

Comment:
I might sound superstitious, but it's may worth moving the mouse out of screen at the end of the each test to avoid odd side-effects from mouse hovering in the next tests.